_topic_maps/_topic_map_microshift.yml

```diff
@@ -0,0 +1,148 @@
+# This configuration file dictates the organization of the topic groups and
+# topics on the main page of the doc site for this branch. Each record
+# consists of the following:
+#
+# ---                                  <= Record delimiter
+# Name: Origin of the Species          <= Display name of topic group
+# Dir:  origin_of_the_species          <= Directory name of topic group
+# Topics:
+#   - Name: The Majestic Marmoset      <= Topic name
+#     File: the_majestic_marmoset      <= Topic file under group dir +/-
+#   - Name: The Curious Crocodile      <= Topic 2 name
+#     File: the_curious_crocodile      <= Topic 2 file
+#   - Name: The Numerous Nematodes     <= Sub-topic group name
+#     Dir: the_numerous_nematodes      <= Sub-topic group dir
+#     Topics:
+#       - Name: The Wily Worm          <= Sub-topic name
+#         File: the_wily_worm          <= Sub-topic file under <group dir>/<subtopic dir>
+#       - Name: The Acrobatic Ascarid  <= Sub-topic 2 name
+#         File: the_acrobatic_ascarid  <= Sub-topic 2 file under <group dir>/<subtopic dir>
+#
+# The ordering of the records in this document determines the ordering of the
+# topic groups and topics on the main page.
+
+---
+Name: About
+Dir: welcome
+Distros: microshift
+Topics:
+- Name: Welcome
+  File: index
+- Name: Legal notice
+  File: legal-notice
+---
+Name: Introduction to MicroShift
+Dir: microshift_architecture
+Distros: microshift
+Topics:
+- Name: Understanding MicroShift
+  File: microshift-understanding
+- Name: MicroShift APIs
+  File: microshift-apis
+---
+# Name: Release notes
+# Dir: release_notes
+# Distros: microshift
+# Topics:
+# - Name: MicroShift 4.12 release notes
+#   File: microshift-4-12-release-notes
+# ---
+Name: Installing
+Dir: microshift_install
+Topics:
+- Name: Installing MicroShift from RPM
+  File: microshift-install-rpm
+- Name: Installing MicroShift in a RHEL for Edge Image
+  File: microshift-install-rhel-for-edge
+---
+Name: API reference
+Dir: rest_api
+Distros: microshift
+Topics:
+- Name: Network APIs
+  Dir: network_apis
+  Topics:
+    # FIXME: need to incorporate the supported k8s APIs
+  - Name: 'Route [route.openshift.io/v1]'
+    File: route-route-openshift-io-v1
+- Name: Security APIs
+  Dir: security_apis
+  Topics:
+  - Name: 'SecurityContextConstraints [security.openshift.io/v1]'
+    File: securitycontextconstraints-security-openshift-io-v1
+---
+Name: CLI tools
+Dir: cli_reference
+Distros: microshift
+Topics:
+- Name: CLI tools overview
+  File: index
+- Name: OpenShift CLI (oc)
+  Dir: openshift_cli
+  Topics:
+  - Name: Getting started with the OpenShift CLI
+    File: getting-started-cli
+  - Name: Configuring the OpenShift CLI
+    File: configuring-cli
+  - Name: Managing CLI profiles
+    File: managing-cli-profiles
+  - Name: Extending the OpenShift CLI with plug-ins
+    File: extending-cli-plugins
+  # FIXME: auto-generated content in the next file includes commands
+  # not supported on MicroShift
```

Comment:
It might be easier to track some of these FIXMEs as Jira tickets. I can open those if that's how you prefer to work.

Reply:
I was going to open these as I go, but either way is fine with me.